QuizGame/README.md

```diff
@@ -0,0 +1,28 @@
+# About project "QuizGame":
+## This is a simple version of quiz written on Python that connects with database.
+
+# How to start. First steps
+    git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+
+    git checkout kulgavykh
+
+    cd quizgame
+
+# Next step is creating Docker container and filling database
+    run init_script.sh
```

Comment:
firstly
`chmod +x init_script.sh`
then
`./init_script.sh`

Reply:
Also, you init_script.sh asks password for db to finish. you can [fix this](https://stackoverflow.com/questions/6523019/postgresql-scripting-psql-execution-with-password) or just change the script logic